server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -238,11 +240,13 @@ public Form<EditTiClientInfoForm> updateClientInfo(
   /**
    * Gets all the TrustedIntermediaryAccount managed by the given TI Group with/without filtering
    *
-   * @param searchParameters - This object contains a nameQuery and/or a dateQuery String. If both
-   *     are empty- an unfiltered list of accounts is returned. If nameQuery is present- a match
-   *     between the Account holder's name and the nameQuery is performed. If dateQuery is present -
-   *     a match between the Account holder's Date of Birth and the dateQuery is performed - the
-   *     matched results are collected and sent as an Immutable List.
+   * @param searchParameters - This object contains a nameQuery, a dayQuery, a monthQuery and a
+   *     yearQuery String. If all are empty, an unfiltered list of accounts is returned. If
+   *     nameQuery is present, a match between the Account holder's name and the nameQuery is
+   *     performed. If dayQuery, monthQuery and yearQuery are present, a match between the Account
+   *     holder's Date of Birth and the date queries is performed. The matched results are collected
+   *     and sent as an Immutable List. If name query is empty and any of the date queries are
+   *     empty, an empty list is returned.
```

Comment:
Is an empty list returned or an unfiltered list?

Reply:
An empty list of clients is returned if the name query is empty and only some of the date queries are empty.  See this test:  https://github.com/civiform/civiform/blob/a8b31f19e072de158bc2df1c5d4f9131def20b76/server/test/services/ti/TrustedIntermediaryServiceTest.java#L375

I'll change the wording of the Javadoc so that it's more clear.

Reply:
Javadoc updated